webpack/assets/javascripts/react_app/stores/StatisticsStore.js

```diff
@@ -0,0 +1,52 @@
+import AppDispatcher from '../dispatcher';
+import ActionTypes from '../constants';
+import AppEventEmitter from './AppEventEmitter';
+
+const _statistics = {};
+
+class StatisticsEventEmitter extends AppEventEmitter {
+  constructor() {
+    super();
+    super.setMaxListeners(20);
```

Comment:
what happens if there are more then 20? (not really worried as we currently have about 10 charts)